### PR TITLE
fixes broken slack command

### PR DESCRIPTION
**Pull request by kenliu**

### CODE REVIEW

**Comment by kenliu** [comment]
@kbaum tagging you

File: app/controllers/leader_board_controller.rb [hunk 1]
```diff
@@ -4,10 +4,24 @@ class LeaderBoardController < ApplicationController
...
     end
   end
 
+  private
+
+  def results
+    LeaderBoard.new(date).results
+  end
+
+  def sprint
```
Comment (kbaum) [inline app/controllers/leader_board_controller.rb]: I moved ```sprint``` and ```date``` to application_controller.

**Comment by kenliu** [comment]
@kbaum ok 2 merge?

**Comment by kenliu** [comment]
BTW Gustavo is going to fix this hound nonsense

**Comment by kenliu** [comment]
@kbaum @guivinicius merge me please!

**Comment by kenliu** [comment]
thanks!

On Tue, Feb 2, 2016 at 11:27 AM, Guilherme Vinicius Moreira <
notifications@github.com> wrote:

> Merged #7 <https://github.com/viewthespace/pr-leaderboard/pull/7>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/viewthespace/pr-leaderboard/pull/7#event-536096131>.
>



-- 
<http://vts.com/>
Ken Liu
VP of Engineering
M (973) 637-0007
E ken@vts.com
<https://www.vts.com/testimonials>
<https://twitter.com/viewthespace>  <https://vimeo.com/viewthespace>
<https://instagram.com/viewthespace>
<https://www.linkedin.com/company/view-the-space>
<https://www.facebook.com/Viewthespace>

**Pull request merged**